static/install

```diff
@@ -74,7 +74,13 @@ dune_tar_uri="https://get.dune.build/$today/$target/$tar_target"
 
 install_dir="$HOME/.dune"
 bin_dir="$install_dir/bin"
-tmp_dir="$install_dir/tmp"
+
+tmp_dir="$(mktemp -d --tmpdir dune-install.XXXXXXXXXX)"
```

Comment:
Is `mktemp` available on any POSIX machine or do we need a fallback?

Reply:
I don't think it is POSIX but the command and options seem well supported. I've replaced `--tmpdir` with `-p` to support OpenBSD and it works on Alpine.

[Source](https://stackoverflow.com/a/2792789).